composer.json

```diff
@@ -30,9 +31,6 @@
       "mlocati/composer-patcher": true,
       "kylekatarnls/update-helper": true
     },
-    "platform": {
-        "php": "7.4.26"
```

Comment:
Any reason to remove this and section instead of updating it to the PHP version actually used in production?
This would solve this issue:
- on the production server we have PHP 8.2.0
- on the development machine we (will) have PHP 8.2.1
- we run `composer update` on the development machine
- composer installs a package version that requires PHP 8.2.1
- the production server gets broken

As a rule of thumb, when I have a `composer.lock` file tracked by git (and used in the production server), I always add the `platform.php` configuration key...